miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -312,4 +320,85 @@ public String processSubmit(@ModelAttribute("run") Run run, ModelMap model, Sess
       throw ex;
     }
   }
+
+  @Value("${miso.run.report.links:}")
+  private String runReportLinksConfigLine;
+
+  private Map<String, String> getRunReportServicesUrls(Run run) {
+    Map<String, String> runReportLinks = new TreeMap<>();
+    if (isStringBlankOrNull(runReportLinksConfigLine) || run.getId() == Run.UNSAVED_ID) {
+      return runReportLinks;
+    }
+
+    // config format: "<PlatformType, PlatformType>:<link text>:<use HTTPS>:<domain>:<port>:<page address>:<run field>"
+    // e.g. "Illumina:Run Report:true:local.domain:8080:/runReport?id=~:id"
+    // `~` is used in the <page address> as a placeholder for where the <run field> goes.
+    // multiple run report links may be comma-separated
+    String[] configStrings = runReportLinksConfigLine.split(",");
+    for (int i = 0; i < configStrings.length; i++) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS135)